packages/themeables/text-align/readme.md

```diff
@@ -0,0 +1,14 @@
+# @themeables/text
+
+> What is a _themeable_? You can find more in the [`@themeables` readme](..)
+
+Properties that the theme function for the `@themeables/text-align` should return:
+
+```ts
+type TThemeableTextAlign = {
```

Comment:
If I close my eyes and think about this API, I have the nagging feeling that it is also useful for Layouts of some sorts, so maybe in the future we can change the name. But I don't have any suggestion right now, so it should be fine.

Reply:
Good to see you here :D Thanks for idea! I'll keep that in mind. Maybe as we move forward it will become more obvious to us how to unify it!